src/getting_started_in_kicad/getting_started_in_kicad.adoc

```diff
@@ -829,13 +831,15 @@ image::images/gsik_tutorial1_080.png[gsik_tutorial1_080_png]
     preferred to actual wires because they make the schematic much
     less messy.
 
-11. Now we will define the edge of the PCB. Select 'Edge.Cuts' from
+11. Now we will define the edge of the PCB. Select 'Edge.Cuts' layer from
```

Comment:
I agree. As it was it is not really wrong but adding the word "layer" is more clear. IMHO of course.